docs/reference-architectures/serverless/web-app.md

```diff
@@ -12,46 +12,34 @@ ms.custom: seodec18, serverless
 
-Functions are executed when an external trigger occurs, such as an HTTP request or a message arriving on a queue. This makes an [event-driven architecture style][event-driven] natural for serverless architectures. To coordinate work between components in the architecture, consider using message brokers or pub/sub patterns. For help choosing between messaging technologies in Azure, see [Choose between Azure services that deliver messages][azure-messaging].
+Functions are executed when an external trigger occurs, such as an HTTP request or a message arriving on a queue. In the case of a serverless web app, functions comprise the API of the application and are triggered by an HTTP request. This is called an [Event Driven Aarchitecture][event-driven]. To coordinate work between components in the architecture, consider using message brokers or pub/sub patterns. For help choosing between messaging technologies in Azure, see [Choose between Azure services that deliver messages][azure-messaging].
 
 ## Architecture
 
 The architecture consists of the following components:
 
-**Blob Storage**. Static web content, such as HTML, CSS, and JavaScript files, are stored in Azure Blob Storage and served to clients by using [static website hosting][static-hosting]. All dynamic interaction happens through JavaScript code making calls to the backend APIs. There is no server-side code to render the web page. Static website hosting supports index documents and custom 404 error pages.
-
-**CDN**. Use [Azure Content Delivery Network][cdn] (CDN) to cache content for lower latency and faster delivery of content, as well as providing an HTTPS endpoint.
+**Blob Storage**. Blob stands for "Binary Large Object". In Azure Storage, a blob is just any type of file. The size does not matter. Static web content, such as HTML, CSS, and JavaScript files are all "blobs". They are stored in Azure Blob Storage and served to clients by using [static website hosting][static-hosting]. All dynamic interaction happens through JavaScript code making calls to the backend APIs. There is no server-side code to render the web page. Static website hosting supports index documents and custom 404 error pages.
 
 **Function Apps**. [Azure Functions][functions] is a serverless compute option. It uses an event-driven model, where a piece of code (a "function") is invoked by a trigger. In this architecture, the function is invoked when a client makes an HTTP request. The request is always routed through an API gateway, described below.
 
-**API Management**. [API Management][apim] provides an API gateway that sits in front of the HTTP function. You can use API Management to publish and manage APIs used by client applications. Using a gateway helps to decouple the front-end application from the back-end APIs. For example, API Management can rewrite URLs, transform requests before they reach the backend, set request or response headers, and so forth.
```

Comment:
I'm recommending that we remove CDN and API Management and point people to Front Door. There are things that API Management does that Front Door does not. In fact, we may want to consider an architecture where we recommend an API Management layer in addition to Front Door.

What are the use cases for need APIM in addition to Front Door? Can we get away with not recommending that as it's an advanced enough use case?